### ISSUE TITLE
Switching the f-droid build to this fork

### ISSUE THREAD

**Issue by Bubu** [rdoeffinger/MapEver#1]
MapEver is currently only in the archive repo at f-droid and there is demand to get this back to the main repo: https://github.com/diedricm/MapEver/issues/23

Even better would be of course to include all the fixes you have done which are not yet included in any release in diedricm/MapEver.

Would it work if I switch the f-droid app to this repo and add your version 1.1.1 as an update to the existing MapEver?

**Comment by rdoeffinger** [comment]
Yes, I think that should be fine, though note that I threw out all the OpenCV stuff, if someone was particularly attached to e.g. the automatic corner detection they might be disappointed (never worked in a useful way for me, and a pain to build and maintain).
That might also mean you don't need submodule support.
I might also make sense to have it automatically update for new tags like the QuickDic build does.

**Comment by Bubu** [comment]
Thanks for the answer.
The old version will still be in the archive if someone wants it. I'll do this in the next couple of days then.

**Comment by Bubu** [comment]
I tried to build this but got the following error:
```
:MapEver:compileReleaseJavaWithJavac/home/mhoffmann/projects/oss/fdroiddata/build/de.hu_berlin.informatik.spws2014.mapever/MapEver/src/de/hu_berlin/informatik/spws2014/mapever/navigation/Navigation.java:497: error: cannot find symbol
                GpsPoint pos = locationDataManager.getGpsPosition(mapView.longClickPos);
                                                  ^
  symbol:   method getGpsPosition(Point2D)
  location: variable locationDataManager of type LocationDataManager
```
Any ideas?
(Running `gradle assembleRelease`)

**Comment by rdoeffinger** [comment]
I was wrong about the submodules.
There is still a submodule for src/de/hu_berlin/informatik/spws2014/ImagePositionLocator.
Your error indicates you have the wrong (and old) version of that submodule.

**Comment by rdoeffinger** [comment]
Hm... Also the submodule URL is wrong, it should have pointed to my fork of ImagePositionLocator.
I didn't want to make a new release yet, so I only made a v1.1.2 tag that is identical except for that fix.
It seemed to work fine like this:
git clone --recurse-submodules git@github.com:rdoeffinger/MapEver.git -b v1.1.2

**Comment by orivej** [comment]
@Bubu Would you package this for F-Droid?

**Comment by davidmef** [comment]
I see that there is a September 2021 version on Google Play. Is the source code in this repo?
Are there obstacles to distribute this with F-Droid as well?
